schedule.html

```diff
@@ -246,7 +265,7 @@ <h2 id="saturday">Saturday 10th June</h2>
         </tr>
         <tr>
             <td>14:50</td>
-            <td colspan="2">
+            <td>
```

Comment:
No it shouldn't be colspan 2 (the track 2 column spans the whole day including lunch & breaks)